src/nvim/ops.c

```diff
@@ -5291,6 +5291,8 @@ static yankreg_T *adjust_clipboard_name(int *name, bool quiet)
 
 static bool get_clipboard(int name, yankreg_T **target, bool quiet)
 {
+  bool noerrmsg = false;
```

Comment:
Why not `errmsg = true`, and reverse the conditions elsewhere? This would avoid the double-negative below:
```
if (!noerrmsg) {
  EMSG("clipboard: provider returned invalid data");
}
```